assets/images/exclamation.svg

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="iso-8859-1"?>

```

Comment:
Can you show me what this graphic looks like? Where is it coming from?

Reply:
This is what it looks like. I had to add it to show warning and error notifications.
![Screen Shot 2021-05-15 at 4 11 02 AM](https://user-images.githubusercontent.com/11542415/118339858-930d1600-b533-11eb-9ad9-8b5025275aae.png)

Reply:
Could you please use this image instead? 
[circle-exclaimation.svg.zip](https://github.com/Expensify/Expensify.cash/files/6481540/circle-exclaimation.svg.zip)

Reply:
Yes. Updated